opentelemetry-sdk/tests/trace/propagation/test_b3_format.py

```diff
@@ -258,3 +258,47 @@ def test_missing_span_id(self):
         ctx = FORMAT.extract(get_as_list, carrier)
         span_context = get_span_from_context(ctx).get_context()
         self.assertEqual(span_context.span_id, trace_api.INVALID_SPAN_ID)
+
+    def test_parent_span_context(self):
```

Comment:
I don't think a new test is required, the current ones have to be fixed https://github.com/open-telemetry/opentelemetry-python/blob/8c360a182d61a8694904bbcc99ef9f60a28e342a/opentelemetry-sdk/tests/trace/propagation/test_b3_format.py#L48. Parent cannot be a `Span`.

Reply:
Fixed